env.d/development/common.dist

```diff
@@ -21,6 +21,7 @@ STORAGES_STATICFILES_BACKEND=django.contrib.staticfiles.storage.StaticFilesStora
 AWS_S3_ENDPOINT_URL=http://minio:9000
 AWS_S3_ACCESS_KEY_ID=impress
 AWS_S3_SECRET_ACCESS_KEY=password
+MEDIA_BASE_URL=http://localhost:8083
```

Comment:
nitpick: technically the previous commit already introduced something about this setting which should only arrive in the commit.